OpenModelica/StartTLMOpenModelica.bat

```diff
@@ -10,7 +10,7 @@ REM 6 model-file
 
 rem *********************SET PATHS****************************************************
 rem set path to OpenModelica installation folder
-set OpenModelicaPath=%OPENMODELICAHOME%
+set "OpenModelicaPath=%OPENMODELICAHOME%"
 
 rem *********************SETUP SIMULATION*********************************************
 set OMC_Cmd=%OpenModelicaPath%/bin/omc.exe

```

Comment:
It doesn't work. Instead change this line to,

`set OMC_Cmd="%OpenModelicaPath%/bin/omc.exe"`

Reply:
Thanks for reviewing. I changed it accordingly.